qa-deploy

```diff
@@ -55,8 +55,12 @@ function add_docker_credentials_microk8s() {
 }
 
 function apply_configuration() {
-    microk8s.kubectl apply -f config.global.yaml --namespace kube-system
     microk8s.kubectl apply -f config.qa.yaml
+
+    # On the version of the ingress controller we have we need to change the
+    # name to be able to apply the configuration. By default the nginx ingress
+    # controller is searching for a config-map called: nginx-load-balancer-conf
+    cat "config.global.yaml" | sed 's|name: nginx-configuration|name: nginx-load-balancer-conf|' | microk8s.kubectl apply --filename - --namespace default
```

Comment:
Didnt you remove `name: nginx-load-balancer-conf`? Why would this still be used?

Reply:
I keep it because on microk8s it's the default name used but not on production

Reply:
yes, unfortunately, the name changed upstream for the default configmap. It is just a command-line parameter to the nginx pods though, so we could make this a config option in cdk or change microk8s or cdk to align again. I would be more a fan of changing microk8s to align with upstream. I added https://github.com/ubuntu/microk8s/issues/291 to try and address this.